python-checks/src/main/java/org/sonar/python/checks/UselessParenthesisCheck.java

```diff
@@ -24,21 +24,29 @@
 import org.sonar.plugins.python.api.tree.Expression;
 import org.sonar.plugins.python.api.tree.ParenthesizedExpression;
 import org.sonar.plugins.python.api.tree.Tree;
+import org.sonar.python.quickfix.IssueWithQuickFix;
+import org.sonar.python.quickfix.PythonQuickFix;
+import org.sonar.python.quickfix.PythonTextEdit;
 
 @Rule(key = UselessParenthesisCheck.CHECK_KEY)
 public class UselessParenthesisCheck extends PythonSubscriptionCheck {
 
   public static final String CHECK_KEY = "S1110";
 
   private static final String MESSAGE = "Remove those useless parentheses.";
+  public static final String QUICK_FIX_MESSAGE = MESSAGE;
```

Comment:
I'd suggest `Remove the redundant parentheses` for the quickfix message.

I also think we should update the regular issue message to `Remove those redundant parentheses` as the word `useless` is quite strong and we should probably favor the word `redundant` instead (this is a general feedback we received in the past, but it seems it was not addressed).

Changing the issue message has other implications, so let's do it in a separate PR.